bayesian/dependency_finder.py

```diff
@@ -46,6 +50,18 @@ def execute(self, arguments, db, manifests):
         # If we receive a manifest file we need to save it first
         result = []
         for manifest in manifests:
+            content_hash = None
+            if source == 'osio':
+                content_hash = generate_content_hash(manifest['content'])
+                print("{} file digest is {}".format(manifest['filename'], content_hash))
+
+                s3 = AmazonS3(bucket_name='boosters-manifest')
+                try:
+                    s3.connect()
+                    manifest['content'] = s3.retrieve_blob(content_hash).decode('utf-8')
+                except ClientError as e:
+                    print("Unexpected error while retrieving S3 data: %s" % e)
```

Comment:
either we can use a logger error or we can `raise FatalTaskError`

Reply:
@miteshvp this is within server. So, raising FatalTaskError does not make sense. Rather any exception here is going to send an HTTP 500 response.